File: 5789.py
from datetime import datetime

class Solution:
    def numberOfRounds(self, startTime: str, finishTime: str) -> int:
        s = datetime.strptime(startTime, "%H:%M")
        e = datetime.strptime(finishTime, "%H:%M")
        ROUND = 15 * 60
        return int(e.timestamp() // ROUND - (s.timestamp() + ROUND - 1) // ROUND) % (24 * 4)

File: 5789.ts
function numberOfRounds(startTime: string, finishTime: string): number {
        const getMinute = (time:string):number => parseInt(time.slice(0, 2)) * 60 + parseInt(time.slice(3));
        const ROUND = 15;
        return (Math.floor(getMinute(finishTime) / ROUND) - Math.ceil(getMinute(startTime) / ROUND) + 24 * 4) % (24 * 4);
};

